lib/node_modules/@stdlib/stats/base/dists/triangular/ctor/benchmark/benchmark.js

```diff
@@ -153,16 +170,21 @@ bench( pkg+'::set:b', function benchmark( bm ) {
 	c = 30.0;
 	dist = new Triangular( a, b, c );
 
+	len = bm.iterations;
```

Comment:
```suggestion
	c = 30.0;
	dist = new Triangular( a, b, c );
	len = 100;
```

No empty line between variable initializations.